lib/awful/layout/init.lua.in

```diff
@@ -109,7 +109,15 @@ function layout.arrange(screen)
     p.geometry = capi.screen[screen].geometry
     p.clients = client.tiled(screen)
     p.screen = screen
-    layout.get(screen).arrange(p)
+
+    -- Apply layout geometry
+    local gs = layout.get(screen).arrange(p)
+    for k, gp in ipairs(gs) do
+        gp[2].height = gp[2].height - gp[1].border_width * 2
+        gp[2].width = gp[2].width - gp[1].border_width * 2
+        gp[1]:geometry(gp[2])
+    end
+
```

Comment:
Can you store this information in the tag. This can be used to "retro generate" a pixmap that represent the layout

The collision plugin currently have a hack to do it:
http://i.imgur.com/VvdgYg7.png

Only store this if only 1 tag is selected